yellowbrick/features/radviz.py

```diff
@@ -159,6 +160,9 @@ def draw(self, X, y, **kwargs):
         draws each instance as a class or target colored point, whose location
         is determined by the feature data set.
         """
+        # Convert from dataframe
+        if is_dataframe(X):
```

Comment:
Scikit-Learn has some utilities called `_check_target` and `_check_instances` that do this kind of conversion. This is just a general comment to remind me to think about how to deal with this in a more meaningful way in the future.